Automattic/vip-go-mu-plugins
# Bump VIP Dashboard to introduce cap filter

https://github.com/Automattic/vip-dashboard/pull/89

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).

## Steps to Test

- Verify that the VIP Page continues to load.
- Then, add the following and verify that only admins can access the VIP Page:

```
add_filter( 'vip_dashboard_page_cap', function( $cap ) {
    // restrict to administrators
    return 'manage_options';
} );
```
